ppl/zqd/core.go

```diff
@@ -159,6 +159,8 @@ func NewCore(ctx context.Context, conf Config) (*Core, error) {
 	case "recruiter":
 		c.workerPool = recruiter.NewWorkerPool()
 		c.addRecruiterRoutes()
+	case "temporal":
+		c.addAPIServerRoutes()
```

Comment:
Why is this necessary?

Reply:
It was necessary for the Ztest "compact" to run. I was expecting that the zqd-temporal workers would need different routes. The existing compact test was working personality "all" and depended on APIServer routes.

Reply:
Please remove this and change the test so `zapi` talks to the API server instead of the Temporal worker.

Reply:
OK, so the APIServer also has access to the Temporal API's for added work to the Temporal queue, and the zqd-temporal worker will only dequeue work items added by the APIServer. Is that correct?

Reply:
Yes.

Reply:
After making this change, the cluster-compact test fails because it is no longer performing compaction. Maybe the root service as it is currently built does not use the new Temporal API?

Reply:
The resolution to this was to remove the APIServerRoutes from the temporal worker, but keep the case statement so the instance would start. In addition, we added temporal.enabled and other startup flags to the root service.